src/loaders/tiff/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface TiffSelection {
```

Comment:
Slightly pedantic, but since both `loadMultiTiff` and `loadOmeTiff` currently both use the OME data model, I would keep this named as `OmeTiffSelection`. Both functions return an identical metadata object which is OME-XML derived.

Reply:
Could we also move this to `lib/utils.ts`? Just want to avoid tons of files.